src/containerapp/azext_containerapp/custom.py

```diff
@@ -274,9 +608,9 @@ def create_containerapp(cmd,
                         min_replicas=None,
                         max_replicas=None,
                         target_port=None,
-                        transport="auto",
+                        transport=None,
                         ingress=None,
-                        revisions_mode="single",
+                        revisions_mode=None,
```

Comment:
May I ask will the modification of the default value here cause breaking change to users?

Reply:
Removed this change